internal/endtoend/testdata/emit_db_tags/go/models.go

```diff
@@ -0,0 +1,14 @@
+// Code generated by sqlc. DO NOT EDIT.
+
+package querytest
+
+import (
+	"database/sql"
+)
+
+type User struct {
+	ID        int32          `db:"id"`
```

Comment:
According to the gorp docs, we could add a bit more information here

```go
type User struct {
    ID         int64     `db:"id, primarykey, autoincrement"`
    ...
}
```

Reply:
1. Can you point me to this in the gorp docs?

2. Does this provide any value? AFAIK other attributes like that would only be used for creating tables based on gorp structs, but if we're using sqlc, the tables must have already been created.

Reply:
There doesn't appear to be any section in the docs. You can find a few different use cases in the [quickstart](https://github.com/go-gorp/gorp#quickstart).

```go
type Post struct {
    // db tag lets you specify the column name if it differs from the struct field
    Id      int64  `db:"post_id"`
    Created int64
    Title   string `db:",size:50"`               // Column size set to 50
    Body    string `db:"article_body,size:1024"` // Set both column name and size
}


// Example of using tags to alias fields to column names
// The 'db' value is the column name
//
// A hyphen will cause gorp to skip this field, similar to the
// Go json package.
//
// This is equivalent to using the ColMap methods:
//
//   table := dbmap.AddTableWithName(Product{}, "product")
//   table.ColMap("Id").Rename("product_id")
//   table.ColMap("Price").Rename("unit_price")
//   table.ColMap("IgnoreMe").SetTransient(true)
//
// You can optionally declare the field to be a primary key and/or autoincrement
//
type Product struct {
    Id         int64     `db:"product_id, primarykey, autoincrement"`
    Price      int64     `db:"unit_price"`
    IgnoreMe   string    `db:"-"`
}
```

Also, there are apparently two versions of grop? v1 and v2?

> Does this provide any value?
I think it does. If a field is marked as primary key, `Insert` can return the ID of the created record.

Reply:
> I think it does. If a field is marked as primary key, Insert can return the ID of the created record.

Ah yeah, that's true. Ok, I can add that then and rename the field to be gorp-specific.

Reply:
The other option, before you get started on the rename, is to have the `emit_db_tags` only emit the column names. Then have a separate `emit_gorp_tags` that does all the magic. My preference is for tool-specific configuration, because I doubt there are too many people using a combination of grop / sqlx / gorm.

Reply:
My preference would probably be to just leave it as the column names and not worry about the other gorp annotation stuff, since that's consistent across gorp/sqlx/gorm, and also gets us most of the value (all of the value for my specific use-case -- we use `.SetKeys()` to explicitly specify the primary keys instead of the `primarykey` tag).

But if we're going to include the extra gorp features now, yeah, I think having a single "gorp" option makes sense.

Reply:
> My preference would probably be to just leave it as the column names and not worry about the other gorp annotation stuff

Great, let's go ahead and keep it as is, naming included. I'll open an issue tracking a future use case for orm-specific tags